modules/clone.sh

```diff
@@ -42,5 +71,11 @@ function clone_help {
   echo "      Will clone the current system to /dev/sdb (by default)."
   echo ""
   echo "  $(basename "$0") clone /dev/sda"
-  echo "      Will clone the current system to /dev/sda"
+  echo "      Will clone the current system to /dev/sda."
+  echo ""
+  echo "  $(basename "$@") clone --reboot"	
```

Comment:
`echo "  $(basename "$@") clone --reboot"` should be
`echo "  $(basename "$0") clone --reboot"`?

Same thing with line 79.

Reply:
Fixed